homework.md

```diff
@@ -44,70 +44,73 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language .
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
 When people want to show off code to each other they can put it on Github
+Git is used to tracking changes in the source code
+It allows multiple developers to work together
+Easy Project Management
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+Fork means to make a copy of the repository
 
 ## Question 10 
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+When you clone a repository, you copy the repository from GitHub to your local machine.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+Git history log shows a list of all the commits made to a repository
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+Git push allows you to send (or push) the commits from your local branch in your local Git repository to the remote repository.
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+Github is a internet host for software development
```

Comment:
I think we can try to go into detail to explain how does it help software development: it is a code hosting platform for version control and collaboration.